src/components/Channel/hooks/useCreateMessagesContext.ts

```diff
@@ -38,9 +37,10 @@ export const useCreateMessagesContext = <
   editMessage,
   FileAttachment,
   FileAttachmentGroup,
+  FileAttachmentIcon,
   flagMessage,
   FlatList,
-  forceAlign,
+  forceAlignMessages,
```

Comment:
forceAlign is in the COOKBOOK, we need to change it there

Reply:
Already did!!